example-config.yml

```diff
@@ -386,6 +386,7 @@ itinerary:
 #   targets:
 #     testTarget: # To set a name for this target, create the i18n string "config.popups.testTarget"
 #       url: http://example.com/ 
+#       modal: false # Whether to always launch the popup as a browser popup
```

Comment:
True = the opposite of what the comment says, which I think is a little confusing right?

Reply:
Yes I can clarify this